plugin/trace/setup.go

```diff
@@ -34,14 +34,30 @@ func setup(c *caddy.Controller) error {
 	return nil
 }
 
+func endPointName(hosts []string, port string) string {
+	host := ""
+	switch len(hosts) {
+	case 0:
+	case 1:
+		host = hosts[0]
+	default:
+		host = hosts[0]
+		for _, h := range hosts[1:] {
+			host = host + "," + h
+		}
+		host = "(" + host + ")"
+	}
+	return host + ":" + port
```

Comment:
This should actually be net.JoinHostPort

Reply:
JoinHostPort does not support Host or IP to be empty. Here we can have host empty.

Reply:
this does the wrong thing for v6 then.. unless `trace` handles that?